src/main/java/appeng/facade/FacadePart.java

```diff
@@ -49,7 +49,7 @@ public ItemStack getItemStack() {
 
     @Override
     public void getBoxes(final IPartCollisionHelper ch, boolean itemEntity) {
-        if (itemEntity || ch.isBBCollision()) {
+        if (itemEntity /* || ch.isBBCollision() */) {
```

Comment:
Don't comment out code like that :-|

Reply:
That is for reference as I have no clue what it does. In case we have to revert it in 3 months. I'll certainly remove it for a stable release.

Reply:
you'll have forgotten about it long before we come to a stable release :-P

It's quite obvious _what_ it does. It doesn't tell us _why_ it does it though.